### PR TITLE
feat: [#187978144] [#188538877] [#188538893] [#188538912]- Employment and Personnel Services License Status

### PR DESCRIPTION
**Pull request by cbrunefearless**
## Description
This ticket covers all four check statuses under the Employment and Personnel Services Industry Section

### Ticket

This pull request resolves [#187978144](https://www.pivotaltracker.com/story/show/187978144).
This pull request resolves [#188538877](https://www.pivotaltracker.com/story/show/188538877).
This pull request resolves [#188538893](https://www.pivotaltracker.com/story/show/188538893).
This pull request resolves [#188538912](https://www.pivotaltracker.com/story/show/188538912).

### Approach
Added licenses for RGB that corresponded to their license type.

| License Type                                                                    | License Task ID                        | Old Renewal Filename | New Renewal Filename               |
|---------------------------------------------------------------------------------|----------------------------------------|----------------------|------------------------------------|
| Employment & Personnel Service-Employment Agency                               | search-licenses-employment-agency      | employment-agency    | employment-agency-license          |
| Consulting Firms/Temporary Help Services-Temporary Help Service                 | temporary-help-service-firm-reg        | employment-agency_2  | temporary-help-service-firm-reg    |
| Consulting Firms/Temporary Help Services-Consulting Firm                        | consulting-firm-headhunter-reg         | employment-agency_3  | consulting-firm-headhunter-reg     |
| Consulting Firms/Temporary Help Services-Consulting Firm/Temporary Help Service | temp-help-consulting-firm-combined-reg | employment-agency_1  | temp-help-consulting-firm-combined |

### Steps to Test
Tested with RGB data and updated the spreadsheet for each.

## Code author checklist

- [X] I have rebased this branch from the latest main branch
- [X] I have performed a self-review of my code
- [X] I have created and/or updated relevant documentation on the engineering documentation website
- [X] I have not used any relative imports
- [X] I have pruned any instances of unused code
- [X] I have not added any markdown to labels, titles and button text in config
- [X] If I added/updated any values in `userData` (including `profileData`, `formationData` etc), then I added a new migration file
- [X] I have checked for and removed instances of unused config from CMS
- [X] If I added any new collections to the CMS config, then I updated the search tool and `cmsCollections.ts` (see CMS Additions in Engineering Reference/FAQ on the engineering documentation site)
- [X] I have updated relevant `.env` values in both `.env-template` and in Bitwarden

### CODE REVIEW

File: content/src/roadmaps/tasks/understand-license-requirements-employment-agency.md [hunk 1]
```diff
@@ -22,4 +22,4 @@ callToActionText: ""
 
 Determined the requirements for your employment and personnel services agency.
 
-:::
\ No newline at end of file
+:::
```
Comment (cbrunefearless) [inline content/src/roadmaps/tasks/understand-license-requirements-employment-agency.md:25]: Linter wants a space in it :(

**Comment by FarazA22** [comment]
I don't see the following DEV note being addressed - Update the renewal ID to match the URL slug. It's referenced in three of the tickets.

**Review by FarazA22 — CHANGES_REQUESTED** [review]
I don't see the following DEV note being addressed - Update the renewal ID to match the URL slug. It's referenced in three of the tickets. You need to update the ID and Filename.

File: content/src/license-calendar-events/employment-agency-license.md [hunk 2]
```diff
@@ -23,4 +23,4 @@ Your employment agency license and agent license expires every year.
 * Applicable fees paid
 
 :::callout{ showHeader="true" headerText="" showIcon="false" calloutType="conditional" }
-A renewed employment agency license and agent license are valid for 1 year. It will be sent to your mailing address on file.
\ No newline at end of file
+A renewed employment agency license and agent license are valid for 1 year. It will be sent to your mailing address on file.
```
Comment (cbrunefearless) [inline content/src/license-calendar-events/employment-agency-license.md:26]: Linter wants a space

**Review by jphechter — APPROVED** [review]
I think this all looks good.

I did have a somewhat difficult time tracking down the changes required for this because the info was spread across 4 tickets. To make it a little easier on myself, I added a table on the PR description under the approach section that consolidates the information in those 4 tickets.

I understand that doing all 4 of these at the same time was relatively easy. No issue with that. But when it comes time to review, make it easy on the folks you tap for reviews.

File: content/src/roadmaps/license-tasks/temp-help-consulting-firm-combined-reg.md [hunk 3]
```diff
@@ -20,7 +20,7 @@ callToActionText: Register My Firm
 callToActionLink: https://www.njconsumeraffairs.gov/regulated/Pages/Regulated-Business-Online-Registration.aspx
 webflowId: 65bd67f3b2eede76015e40e3
 licenseCertificationClassification: undefined
-licenseName: ""
```
Comment (FarazA22) [inline content/src/roadmaps/license-tasks/temp-help-consulting-firm-combined-reg.md:23]: update url slug?

File: content/src/roadmaps/tasks/understand-license-requirements-employment-agency.md (hunk 1 above)
Comment (FarazA22) [reply content/src/roadmaps/tasks/understand-license-requirements-employment-agency.md:26]: I don't see any changes made by you in this file, I recommend rolling back the changes / ignore this file from the commit.

**Review by FarazA22 — COMMENTED** [review]
Two minor changes, the rest looks good

File: content/src/roadmaps/tasks/understand-license-requirements-employment-agency.md (hunk 1 above)
Comment (cbrunefearless) [reply content/src/roadmaps/tasks/understand-license-requirements-employment-agency.md:26]: yes sir

File: content/src/roadmaps/license-tasks/temp-help-consulting-firm-combined-reg.md (hunk 3 above)
Comment (cbrunefearless) [reply content/src/roadmaps/license-tasks/temp-help-consulting-firm-combined-reg.md:23]: done

**Pull request merged**